### ISSUE TITLE
Add Article Stubs (Glossary)

### ISSUE THREAD

**Issue by Pennek** [ppy/osu-wiki#3824]
See the main issue #3143
See the project: https://github.com/ppy/osu-wiki/projects/3

This issue aims to split the 115 entries in the Glossary article into stubs/articles of their own and tracks the status of this task.

- [x] A-lettered articles (6) - #3488
- [x] B-lettered articles (13) - #3488
- [x] C-lettered articles (13) - #3505
- [x] D-lettered articles (4) - #3821
- [x] E-lettered articles (1) - #3821
- [x] F-lettered articles (3) - #3827
- [x] G-lettered articles (5) - #3827
- [x] H-lettered articles (2) - #3828
- [x] I-lettered articles (1) - #3828
- [x] J-lettered articles (1) - #3828
- [x] K-lettered articles (4) - #3828
- [x] L-lettered articles (5) - #3840
- [x] M-lettered articles (5) - #3840
- [x] N-lettered articles (3) - #3844
- [x] O-lettered articles (7) - #3859
- [x] P-lettered articles (5) - #3867
- [x] Q-lettered articles (2) - #3867
- [x] R-lettered articles (6) - #3867
- [x] S-lettered articles (15) - #3897
- [x] T-lettered articles (9) - #4698
- [x] U-lettered articles (2) - #3880
- [x] V-lettered articles (1) - #3880
- [x] W-lettered articles (1) - #3880
- [x] Z-lettered articles (1) - #3880

The glossary article can be removed once every entry listed has stubs that are tagged, organized and cross-referenced well. Before the glossary is removed completely we should also double-check that nothing is missing.

**Comment by cl8n** [comment]
> The glossary article can be removed once every entry listed here has a stub

and let's say all of the new ones should be tagged and referenced well too. I wouldn't feel good about deleting an important article like Glossary without knowing that its purpose is entirely replaced. might also depend on #2606 + organizing categories better because it's difficult to discover these articles if you don't already know about them

**Comment by Pennek** [comment]
Agree, articles need better categorisation/organisation. I've been focusing my attention on writing content and ensuring as little information as possible gets lost between the conversion from glossary to stubs more so than categorising everything correctly

**Comment by paper-toaster** [comment]
@Pennek dibs on letters U–Z

**Comment by paper-toaster** [comment]
Has no one taken the T-lettered stubs or is it just not listed yet?

**Comment by TPGPL** [comment]
Looks like they havent been taken yet

**Comment by paper-toaster** [comment]
dibs on the t-lettered articles then

**Comment by Joehuu** [comment]
This seems to be done (i.e. everything is checked), so closing.

**Comment by TicClick** [comment]
> This seems to be done (i.e. everything is checked), so closing.

it needs a follow-up issue, because
1. the glossary is still in place
2. a lot of redirects point to it
3. a lot of articles still point to the glossary
